### ISSUE TITLE
The loss does not decrease

### ISSUE THREAD

**Issue by zhenyuZ-HUST** [VITA-Group/layerGraftedPretraining_ICLR23#1]
When the model was pre-trained on imageNet, the loss function value remained around 3.55 without decreasing.

**Comment by geekJZY** [comment]
Hi,

Can you provide more details about your question? Like how to reproduce your problem? What are the cmds you are running? What's the setting of your problem?

Thanks,
Ziyu Jiang
